api/src/main/java/org/openmrs/Condition.java

```diff
@@ -136,6 +150,43 @@ public static Condition copy(Condition fromCondition, Condition toCondition) {
 		toCondition.setDateVoided(fromCondition.getDateVoided());
 		return toCondition;
 	}
+
+	/**
+	 * Compares properties with those in the given Condition to determine if they have the same meaning
+	 * This method will return true immediately following the creation of a Condition from another Condition
+	 * @see Condition#newInstance(Condition)
+	 * This method will return false if any value is different, excepting identity data (id, uuid)
+	 * If the given instance is null, this will return false
+	 * @param c the Condition to compare against 	
+	 * @return true if the given Condition has the same meaningful properties as the passed Condition
+	 * @since 2.6.1
+	 */
+	public boolean matches(Condition c) {
```

Comment:
Annoying, but would good to have a couple quick test cases around this.

Reply:
Yeah

Reply:
Added, see comment below @mogoodrich